tree/dataframe/test/dataframe_simple.cxx

```diff
@@ -841,27 +838,40 @@ TEST_P(RDFSimpleTests, HistosOneWeightPerEvent)
 {
    using floats = std::vector<float>;
    auto df = RDataFrame(1);
-   auto d = df.Define("v0", [](){floats v({1,2,3});return v;})
-              .Define("v1", [](){floats v({4,5,6});return v;})
-              .Define("v2", [](){floats v({7,8,9});return v;})
-              .Define("w",[](){return 3;});
+   auto d = df.Define("v0",
```

Comment:
Because in this one case, the code is short and regular, I find the original actually more readable 
```suggestion
   // clang-format off
   auto d = df.Define("v0", [](){ floats v({1,2,3}); return v; })
              .Define("v1", [](){ floats v({4,5,6}); return v; })
              .Define("v2", [](){ floats v({7,8,9}); return v; })
              .Define("w",  [](){ return 3; });
   // clang-format on
```
There is a similar set on (new) line 927 where the better choice is not as clear since the lamdba code is not as regular and is longer.